Fix a build error "narrowing conversion from int to char inside { }".
This breaks the build with some versions of g++.

See dyninst commit:

commit e90b6e3be40b63295a462446396d4a1e87b809b2
Author: John Detter <jdetter@wisc.edu>
Date:   Mon Jun 27 18:47:43 2016 -0500


diff -Naurb dyninst.orig/common/src/arch-x86.C dyninst/common/src/arch-x86.C
--- dyninst.orig/common/src/arch-x86.C	2016-10-04 15:14:57.250275480 -0500
+++ dyninst/common/src/arch-x86.C	2016-10-04 15:15:44.128970166 -0500
@@ -511,7 +511,7 @@
 #define VEXW_MAX VEXW92
 
 /* SIMD op conversion table */
-static char vex3_simdop_convert[3][4] = {
+static int vex3_simdop_convert[3][4] = {
   {0, 2,  1, 3},
   {0, 2,  1, 3},
   {0, 1, -1, 2}
